components/Navbar/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from "react";
+import {
+  Box,
+  useColorModeValue,
+  Drawer,
+  DrawerContent,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { MobileNav } from "../MobileNav";
+import { SidebarContent } from "../Sidebar";
+
+/**
+ *
+ * @return {box}
+ */
+export default function Navbar({ children }: { children: ReactNode }) {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  return (
+    <Box>
+      <Box minH="100vh" bg={useColorModeValue("white", "white")}>
+        <SidebarContent
+          onClose={() => onClose}
```

Comment:
Não sei se teve algum motivo em específico para passar para o `onClose` um protótipo de uma função que retorna um protótipo da função `onClose`. Se não teve nenhum motivo em específico, poderia passar `onClose` direto

```suggestion
          onClose={onClose}
```